### PR TITLE
Modified the .bashrc information page

### PR DESCRIPTION
**Pull request by marianalevineu**
Changes that were made:
1. Changed index.rst - moved the bashrc page to the first_steps section.
2. bashrc.rst - modified intro to .bashrc , and titles. 
3. Added a conda section with warning about OOD.

### CODE REVIEW

File: docs/source/first_steps/bashrc.rst [hunk 1]
```diff
@@ -7,13 +7,12 @@ Shell environment on Discovery
 The Discovery Shell environment and .bashrc
 +++++++++++++++++++++++++++++++++++++++++++
 
-Discovery uses a Linux-based Operating System (CentOS) that uses the Shell to interface with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations whichn is the default shell on Discovery.  
+Discovery uses a Linux-based Operating System (CentOS) that uses the Shell to interface with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations which is the default Shell on Discovery.  
 
-The Shell script .bashrc is used by Bash to initialize your Shell environment. For example, it can be used to define shell environemnt variables (such as `PATH`) or aliases and load modules.
+The Shell script .bashrc is used by Bash to initialize your Shell environment. For example, it can be used to define shell environemnt variables (such as ``PATH``) or aliases and load modules.
```
Comment (dhml) [inline docs/source/first_steps/bashrc.rst:12]: Environment variables should generally go in `~/.bash_profile` or `~/.profile`.  `~/.bashrc` is primarily for aliases and functions.

File: docs/source/first_steps/bashrc.rst [hunk 2]
```diff
@@ -7,13 +7,12 @@ Shell environment on Discovery
...
 Your .bashrc file lives in your /home directory. You can make changes to it with a text editor like `nano <https://www.nano-editor.org/>`_.
-Adding sections to your .bashrc file, such as adding modules to load when you sign in or assigning aliases for commands that you use frequently, can be time savers for your workflows.
 
-.. caution:: Making edits to your .bashrc file can result in issues, such as not able to launch an app or not able to execute commands. Incorrect changes to your `PATH` variable may result in the inavibility to use basic Shell commands.  
+.. caution:: Making edits to your .bashrc file can result in many issues. Some changes may prevent you from launching apps or executing commands. Modifying your ``PATH`` variable may result in the inavibility to use basic Shell commands (such as ``cd`` or ``ls``) if not done correctly.
```
Comment (dhml) [inline docs/source/first_steps/bashrc.rst:15]: `inavibility` should be `inability`.

File: docs/source/first_steps/bashrc.rst [hunk 3]
```diff
@@ -27,7 +26,7 @@ You have a default .bashrc file in your home directory when your account is crea
 .. image:: /images/catbashrc.jpg
 
 .. important::
-  We recommend to keep .bashrc unchanged when using Discovery. You can source environment Shell scripts or load modules directly inside your jobs instead. This apprach can prevent potential runtime errors from loading incompatible modules, setting environment variables incorrectly, or from mixing multiple software environments. 
+  We recommend to keep .bashrc unchanged when using Discovery. You can source environment Shell scripts or load modules directly inside your job instead. This apprach can prevent some runtime errors from loading incompatible modules, setting environment variables incorrectly, or from mixing multiple software and Conda environments.
```
Comment (dhml) [inline docs/source/first_steps/bashrc.rst:29]: `apprach` should be `approach`.

File: docs/source/first_steps/bashrc.rst [hunk 4]
```diff
@@ -37,7 +36,7 @@ follow the procedure outlined in :ref:`mini_conda`, there will be a section adde
 
 .. image:: /images/minicondabashrc.jpg
 
-You should not modify this section in the .bashrc file directly. If it was change, remove this section manually using a file editor.
+You should not modify this section in the .bashrc file directly. If it was changed, remove this section manually using a file editor.
 
 .. caution:: We recommend removing the conda initialize section from your .bashrc as it may interfere with the correct startup environment when using Open OnDemand apps. You should always load your Conda environment after your job already started.
```
Comment (dhml) [inline docs/source/first_steps/bashrc.rst:41]: `initialize` should be `initialization`.

File: docs/source/software/conda.rst [hunk 5]
```diff
@@ -40,9 +40,9 @@ Miniconda3 with Python 3.7.
 4. Type ``bash Miniconda3-latest-Linux-x86_64.sh`` to start the installation.
```
Comment (dhml) [inline docs/source/software/conda.rst:40]: We should change `bash Miniconda3-latest-Linux-x86_64.sh` to batch mode installation `bash Miniconda3-latest-Linux-x86_64.sh -b`.  This prevents modification of `~/.bashrc` and makes steps 5 and 6 unnecessary.

**Comment by marianalevineu** [comment]
@dhml  thanks for the suggestions, I applied all of them so please review again at your convenience. Thanks!

**Comment by dhml** [comment]
The file `docs/source/first_steps/.cps_ood.rst.swp` should be removed.

**Comment by marianalevineu** [comment]
@dhml thanks I also removed the file `docs/source/first_steps/.cps_ood.rst.swp`.

**Review by dhml — APPROVED** [review]
@marianalevineu, looks good, thanks!

**Pull request merged**